style.css

```diff
@@ -219,3 +229,52 @@ p.description {
   color: #253858;
   padding-top: 15px;
 }
+
+.my-form {
+  padding-top: 10vh;
+  padding-bottom: 5vh;
+  background-image: url("./images/contactbg.png");
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 100%;
```

Comment:
- [ ]  Hey, the background image is not showing completely and it's missing the black horizontal line at the bottom of the design as seen in the Figma template. please, let's fix this issue. see the images below:

*your design*
![Screenshot from 2023-02-21 11-49-42](https://user-images.githubusercontent.com/59397159/220325054-81f47238-df70-4df3-a02e-15ad5011a646.png)

*Figma design*
![Screenshot from 2023-02-21 11-50-33](https://user-images.githubusercontent.com/59397159/220325044-922e0038-aa2a-4a39-b6c5-e4101938ffbc.png)

Reply:
Thank you Once again! the black bar above represents a smartphone view of the page and is defined by the mobile maker! Though I can add it to make my project approved but it really represents the home button on smartphone and figma designers add it for the purpose of clarity

Reply:
- [ ] Let's work more on the placement of background-image as pointed earlier. You are almost there 👍 I think you need to reduce the background-size and position it to the right. 
```suggestion
  background-image: url("./images/contactbg.png");
  background-color: #6070ff;
  background-repeat: no-repeat;
  background-size: 55%;
  background-position: right 21px;
```